src/main/java/FormController.java

```diff
@@ -8,21 +7,27 @@
 
 public class FormController implements Controller {
     private String resourcePath;
+    private String[] methodsAllowed;
+    private final String NEWLINE = "\r" + System.lineSeparator();
+    private final String REQUEST_SUCCEEDED = "HTTP/1.1 200 OK";
```

Comment:
Also if you want to make these class constants, you'd have to make them `private static final`